tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/Graph/Graph.h"
+#include "glow/IR/IR.h"
+#include "glow/Importer/Caffe2.h"
+#include "gtest/gtest.h"
+
+using namespace glow;
+
+void getImageData(Tensor *result, size_t n, size_t c, size_t h, size_t w) {
+  result->reset(ElemKind::FloatTy, {n, c, h, w});
+  auto RH = result->getHandle<>();
+  for (size_t i = 0, e = n * c * h * w; i < e; i++)
+    RH.raw(i) = i;
+}
+
+/// Test loading conv op from a Caffe2 model.
+/// The input is N*C*H*W (1*1*3*3), the kernel is 2,
+/// stride is 1, pad is 1, group is 1.
+TEST(caffe2, importConv) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename("tests/models/caffe2Models/predict_net.pbtxt");
+  std::string NetWeightFilename("tests/models/caffe2Models/init_net.pbtxt");
+
+  // Make sure to destroy loader as soon as Graph is loaded.
```

Comment:
can you expand this comment as to why we are doing so.

Reply:
Our image classifier and previous testing sample works in this way. Since the loader is not used anymore, it is reasonable to to free the memory.